train_gridworld.py

```diff
@@ -0,0 +1,62 @@
+import random
+import numpy as np
+import matplotlib.pyplot as plt
+from discrete_pendulum import Pendulum
+# plt.style.use('seaborn')
+import gridworld
+import model_based
+import model_free
+
+
```

Comment:
It seems like the only thing missing is the plots of varied epsilon/alpha for Qlearning and SARSA